.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: python
+python:
+  - "3.5"
+install:
+  - pip install -r requirements.txt
+script: nosetests
```

Comment:
Pick a linter you prefer and add that. Document it somewhere and we can get people to switch to it.